src/components/molecules/LogOutput/FullscreenLogOutput/FullScreenLogOutput.tsx

```diff
@@ -12,7 +13,7 @@ import {useCountLines, useLastLines} from '../utils';
 import {StyledFullscreenLogOutputContainer} from './FullscreenLogOutput.styled';
 
 const FullScreenLogOutput = forwardRef<HTMLDivElement, Pick<LogOutputProps, 'actions' | 'logOutput' | 'initialLines'>>(
-  (props, ref) => {
+  (props, ref: any) => {
```

Comment:
Maybe instead of `any` here, in the body use `const containerRef = ref as MutableRefObject<HTMLDivElement | null>`. Btw, did you double check if it's not a function? I don't remember what is the `ref` behavior there, while:

```ts
type ForwardedRef<T> = ((instance: T | null) => void) | MutableRefObject<T | null> | null;
```